src/gpl/src/replace.tcl

```diff
@@ -98,7 +99,18 @@ proc global_placement { args } {
     if { [get_libs -quiet "*"] == {} } {
       utl::error GPL 121 "No liberty libraries found."
     }
+
+    if { [info exists keys(-timing_driven_net_reweight_iter)] } {
+      set overflow_list $keys(-timing_driven_net_reweight_iter)
+    } else {
+      set overflow_list [list 79 64 49 29 21 15]
+    }
```

Comment:
The user list of iters needs to be sorted as there is an implied assumption of decreasing order.

Reply:
Thanks for pointing it out. I've added sort function in C++, so a user doesn't need to sort the overflows.